core/vm/absint_cfg_proof_gen.go

```diff
@@ -87,7 +87,7 @@ func toProgram(code []byte) *Program {
 		op := OpCode(code[pc])
 		stmt.opcode = op
 		stmt.operation = jt[op]
-		stmt.ends = stmt.operation == nil || stmt.operation.halts || stmt.operation.reverts
```

Comment:
This removal has to be verified

Reply:
Ditto https://github.com/ETCCooperative/erigon/pull/4#discussion_r1037179242